src/features/island/hud/components/settings-menu/general-settings/LanguageChangeModal.tsx

```diff
@@ -131,22 +116,31 @@ export const LanguageSwitcher: React.FC<Props> = ({ isOpen, onClose }) => {
             </Modal>
           </span>
         </div>
-      </CloseButtonPanel>
+      </>
     );
   };
 
-  // Close Modal on Hide
-  // eslint-disable-next-line unused-imports/no-unused-vars, @typescript-eslint/no-unused-vars
-  const [view, setView] = useState<"settings">("settings");
+  return Content();
```

Comment:
Can you also change this for the code inside of `Content` and remove the `Content` fn.